File: daeun/sort/K번째수.js
function solution(array, commands) {
  var answer = [];

  for (let i = 0; i < commands.length; i++) {
    let list = array.slice(commands[i][0] - 1, commands[i][1]);
    list.sort((a, b) => a - b);

    answer.push(list[commands[i][2] - 1]); //정렬된 list에서 k번째 값을 answer 배열에 추가
  }

  return answer;
}

File: d15m4s/stack_queue/K번째 수.java

import java.util.Arrays;

public class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int[] answer = new int[commands.length];

        for(int i = 0; i < commands.length; ++i){
            int start    = commands[i][0] - 1;
            int end      = commands[i][1];
            int location = commands[i][2] - 1;
            int[] slice = Arrays.copyOfRange(array, start, end);
            Arrays.sort(slice);
            answer[i] = slice[location];
        }
        return answer;
    }
}
